pkg/core/xds/metadata_test.go

```diff
@@ -98,14 +91,12 @@ var _ = Describe("DataplaneMetadataFromXdsMetadata", func() {
 				},
 			},
 			expected: xds.DataplaneMetadata{
-				AccessLogSocketPath: "/tmp/kuma-al-dp-1-mesh.sock",
-				MetricsSocketPath:   "/tmp/kuma-mh-dp-1-mesh.sock",
-				DynamicMetadata:     map[string]string{},
+				DynamicMetadata: map[string]string{},
 			},
 		}),
 	)
 
-	It("should fallback to service side generated paths", func() { // remove with https://github.com/kumahq/kuma/issues/7220
+	It("should fallback to service side generated paths", func() {
```

Comment:
test names do not make sense anymore. We just removed fallback to have service generated paths. Now we always take paths provided from dpp, right?

I think `should fallback to service side generated paths` is redundant because it's already covered by `Entry("from non-empty node"` and `should fallback to service side generated paths without dpp in metadata` is redundant because it's covered by `Entry("from empty node"`

Reply:
Yeah, you're correct. Thanks for pointing out the deficiency :)